backend/graminsta/graminsta/settings.py

```diff
@@ -37,9 +37,17 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'core'
+    'rest_framework',
+    'rest_framework.authtoken',
```

Comment:
Add blank lines to separate

django apps

third_party apps

graminsta apps